app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<%= render @user %>
+
+<h1 class="text-center m-5">Posts</h1>
+
+<% if !@user.posts.empty? %>
+  <ul class="col p-0">
+    <%= render @user.posts, locals: {truncate: true} %>
+  </ul>
+<% else %>
+  <p class="text-center m-5">There are no posts yet</p>
+<% end %>
```

Comment:
- [ ] kindly fix the show all posts view where I'm getting this error and make sure it matches the wireframe 🙏 


error: 

![image](https://user-images.githubusercontent.com/89282221/222141186-1a381acf-4b67-49e7-8e67-21aa932bcc11.png)



wireframe: 
![image](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABX6AAZP64PKFYHAIUQDGJQUY77KTXA)

Reply:
- [ ] @frnega as previously requested kindly fix the error as shown in the screenshot below, also ensure the page design matches the design given
![image](https://user-images.githubusercontent.com/61408860/222530390-eeffab92-9402-47c2-ac8b-c3e4a0466dbd.png)